app/collins/solr/SolrConfig.scala

```diff
@@ -32,8 +32,9 @@ object SolrConfig extends Configurable {
   def socketTimeout = getInt("socketTimeout", 1000)
   def connectionTimeout = getInt("connectionTimeout", 5000)
   def maxTotalConnections = getInt("maxTotalConnections", 100)
+  def commitWithin = getInt("commitWithinMs", 50)
   def defaultMaxConnectionsPerHost = getInt("defaultMaxConnectionsPerHost", 100)
-  def assetBatchUpdateWindow = getInt("assetBatchUpdateWindowMs", 10) milliseconds
+  def assetBatchUpdateWindow = getInt("assetBatchUpdateWindowMs", 30) milliseconds
```

Comment:
same place the first numbers came from (the ether). its discussed in the description of the PR why these numbers were chosen as a default